packages/react-ui/src/app/connections/create-edit-connection-dialog.tsx

```diff
@@ -187,22 +230,25 @@ const CreateOrEditConnectionDialog = React.memo(
                   <OAuth2ConnectionSettings
                     authProperty={piece.auth as OAuth2Property<OAuth2Props>}
                     piece={piece}
+                    reconnectConnection={reconnectConnection}
```

Comment:
You only need reconnectConnection name not the whole object, can you please change it to the name only

Reply:
I need it to know the connection type for the oauth dialog.